packages/frontend/src/components/PinnedItemsPanel/index.tsx

```diff
@@ -50,6 +50,7 @@ const PinnedItemsPanel: FC<Props> = ({
         <ResourceView
             items={pinnedItems}
             view={ResourceViewType.GRID}
+            isReorderDisabled={false}
```

Comment:
nit: but you can make this hasReorder (positive, instead of negative) and you don't have to pass `={true}` to it. you can make hasReorder prop default to false inside `ResourceView`